plugins/angular.js

```diff
@@ -14,8 +14,33 @@ function angularPlugin(Raven, angular) {
 
     if (!angular) return;
 
+    var _development = false;
+
     function RavenProvider() {
-        this.$get = ['$window', function($window) {
+        Raven.setDataCallback(function(data, original) {
+            angularPlugin._normalizeData(data);
+
+            original && original(data);
+        });
+
+        this.setTagsContext = function(opts) {
```

Comment:
Not have to, but set methods that are like `bootstrap` setters is wise to expose here.

Since `$get` exposes Raven, users still have full access to Raven but not during config phase

I added `setTagsContext` because I needed it by the time of writing, but later on i figured out that its possible to add tags with `config` function.